src/vulnpy/templates/fragments/base.html

```diff
@@ -17,6 +17,23 @@
           <div class="main">
             <ul class="menu-list">
               <li><a href="/vulnpy/cmdi/">CMD Injection</a></li>
+              <li><a href="/vulnpy/deserialization/">Untrusted Deserialization</a></li>
```

Comment:
sorry to keep harping on this. I think the only change we want here is just adding one line for the new list item. There should only be one injection site in the file. The HTML does appear to render properly in the browser, but honestly I'm not sure